identify/extensions.py

```diff
@@ -182,6 +182,7 @@
     'sh': {'text', 'shell'},
     'sln': {'text', 'sln'},
     'sls': {'text', 'salt'},
+    'smk': {'text', 'snakefile'},
```

Comment:
googling "smk extension" doesn't lead to snakefile unfortunately -- so I'm not sure we can accept this :(